python-backend/app/__init__.py

```diff
@@ -45,15 +46,18 @@ def register_extensions(app):
     db.init_app(app)
     jwt.init_app(app)
     apm.init_app(app) if app.config['ELASTIC_ENABLED'] == '1' else None
+    sched.init_app(app)
 
     CORS(app)
     Compress(app)
 
-    if app.config.get('ENVIRONMENT_NAME') == 'prod':
-        sched.init_app(app)
+    if app.config.get('ENVIRONMENT_NAME') == 'test' or app.config.get('ENVIRONMENT_NAME') == 'prod':
         if not app.debug or os.environ.get("WERKZEUG_RUN_MAIN") == 'true':
             sched.start()
             _schedule_NRIS_jobs(app)
+            #This is her to prevent this from running in production until we are confident in the permit data.
```

Comment:
typo in comment and the line below should just not be there if we're not running it 🤷‍♂️ It's not being controlled by a flag or anything so im not certain why the always failing if is there

Reply:
Type aside, this comment is here to only prevent the ETL script from running nightly in production until we are confident in the permit changes and the information that is being brought in.

Reply:
Oh I understand the comment. What I was trying to ask is why is there a 
```
            if False:
                _schedule_ETL_jobs(app)
```

Where you can have no code in there if the code block never runs.
Or even just a 
```
# Comment describing line below
# _schedule_ETL_jobs(app)
```

Just bringing it up because its just a redundant code cycle check and may be picked up as sonarqube as code smell

Reply:
Yeah, not an ideal situation but this code is what will be run when we are more comfortable with all of the permit and permitee information we are expecting and want to see. The if false was chosen as it was thought to look better than commented out code.